### ISSUE TITLE
[Bug]: Combobox options not announced by screen readers

### ISSUE THREAD

**Issue by abh1nash** [radix-vue/radix-vue#1001]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://www.radix-vue.com/components/combobox

### Steps to reproduce

1. Keep your screen reader on
2. Try navigating through the combobox options using arrow keys or home/end
3. None of the options are announced

### Describe the bug

None of the combobox options are announced by screen readers. This is due to the fact that the combobox component doesn't move focus to the option. It merely provides a visual feedback based on data attribute updates - but navigating with arrow keys doesn't move focus to the option.

### Expected behavior

Focus should be moved to the option that is being navigated through.
